scripts/version-logging

```diff
@@ -0,0 +1,2 @@
+LOGGING_VERSION=100.1.2+up3.17.7
```

Comment:
Please double check, upstream rancher has `100.1.2+up3.17.4` and `100.1.3+up3.17.7`

https://github.com/rancher/charts/tree/dev-v2.6/charts/rancher-logging/100.1.2%2Bup3.17.4
https://github.com/rancher/charts/tree/dev-v2.6/charts/rancher-logging/100.1.3%2Bup3.17.7

Reply:
Yes both are in the rancher helm repo as of rancher 2.6.7